packages/edit-site/src/components/post-fields/index.js

```diff
@@ -235,7 +235,7 @@ function usePostFields( viewType ) {
 			{
 				label: __( 'Author' ),
 				id: 'author',
-				getValue: ( { item } ) => item._embedded?.author[ 0 ]?.name,
+				type: 'integer',
```

Comment:
If you remove the elements from the author definition, you can actually edit the author by adding a number :)

Reply:
As I said above, I feel like it's better to actually switch the "type" to "enum" for this case. rather than having to remove "elements"

Reply:
> rather than having to remove "elements"

There's some confusion here: we don't need to remove elements. My point was that if you _were_ to remove elements, the quick edit would still work: the author field would render a number control and you could actually provide the author id and everything would work the same. My point was that the underlying type was `integer` and that select was a _user experience_ improvement.

Reply:
Yes, I know but my thinking was that in order to switch how something renders, I prefer if the change is to actually change "type" or "control" or something like that. It's fine though we can continue this discussion in the other thread.